### ISSUE TITLE
error: Could not enrich AST: MissingBindingKind

### ISSUE THREAD

**Issue by arai-a** [binast/binjs-ref#409]
https://bugzilla.mozilla.org/show_bug.cgi?id=1230194

cannot encode the follow file with --lazy flag.
https://hg.mozilla.org/mozilla-central/raw-file/tip/js/src/jit-test/tests/debug/Object-apply-01.js

```
thread 'large stack dedicated thread' panicked at 'Could not enrich AST: MissingBindingKind', src/libcore/result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
thread 'main' panicked at 'Error in dedicated thread: Any', src/libcore/result.rs:997:5
```

**Comment by arai-a** [comment]
regression from 647ac245705722d61d76fcb32bded91522c64c6d

**Comment by Yoric** [comment]
I believe that this is fixed by #407.

**Comment by RReverser** [comment]
Yeah, duplicate of #406 (exactly same issue).
